arch/risc-v/src/esp32c3/hardware/esp32c3_system.h

```diff
@@ -0,0 +1,1213 @@
+
+/* SYSTEM_EDMA_RESET : R/W [1] : 1'b0 ; */
+
+#define SYSTEM_EDMA_RESET  (BIT(1))
+#define SYSTEM_EDMA_RESET_M  (BIT(1))
+#define SYSTEM_EDMA_RESET_V  0x1
+#define SYSTEM_EDMA_RESET_S  1
+
+/* SYSTEM_EDMA_CLK_ON : R/W [0] : 1'b1 ; */
+
+#define SYSTEM_EDMA_CLK_ON  (BIT(0))
+#define SYSTEM_EDMA_CLK_ON_M  (BIT(0))
+#define SYSTEM_EDMA_CLK_ON_V  0x1
+#define SYSTEM_EDMA_CLK_ON_S  0
+
+#define SYSTEM_CACHE_CONTROL_REG          (DR_REG_SYSTEM_BASE + 0x040)
+
+/* SYSTEM_DCACHE_RESET : R/W [3] : 1'b0 ; */
+
+#define SYSTEM_DCACHE_RESET  (BIT(3))
+#define SYSTEM_DCACHE_RESET_M  (BIT(3))
+#define SYSTEM_DCACHE_RESET_V  0x1
+#define SYSTEM_DCACHE_RESET_S  3
+
+/* SYSTEM_DCACHE_CLK_ON : R/W [2] : 1'b1 ; */
+
+#define SYSTEM_DCACHE_CLK_ON  (BIT(2))
+#define SYSTEM_DCACHE_CLK_ON_M  (BIT(2))
+#define SYSTEM_DCACHE_CLK_ON_V  0x1
+#define SYSTEM_DCACHE_CLK_ON_S  2
+
+/* SYSTEM_ICACHE_RESET : R/W [1] : 1'b0 ; */
+
+#define SYSTEM_ICACHE_RESET  (BIT(1))
+#define SYSTEM_ICACHE_RESET_M  (BIT(1))
+#define SYSTEM_ICACHE_RESET_V  0x1
+#define SYSTEM_ICACHE_RESET_S  1
+
+/* SYSTEM_ICACHE_CLK_ON : R/W [0] : 1'b1 ; */
+
+#define SYSTEM_ICACHE_CLK_ON  (BIT(0))
+#define SYSTEM_ICACHE_CLK_ON_M  (BIT(0))
+#define SYSTEM_ICACHE_CLK_ON_V  0x1
+#define SYSTEM_ICACHE_CLK_ON_S  0
+
+#define SYSTEM_EXTERNAL_DEVICE_ENCRYPT_DECRYPT_CONTROL_REG          (DR_REG_SYSTEM_BASE + 0x044)
+
+/* SYSTEM_ENABLE_DOWNLOAD_MANUAL_ENCRYPT : R/W [3] : 1'b0 ; */
+
+#define SYSTEM_ENABLE_DOWNLOAD_MANUAL_ENCRYPT  (BIT(3))
+#define SYSTEM_ENABLE_DOWNLOAD_MANUAL_ENCRYPT_M  (BIT(3))
+#define SYSTEM_ENABLE_DOWNLOAD_MANUAL_ENCRYPT_V  0x1
+#define SYSTEM_ENABLE_DOWNLOAD_MANUAL_ENCRYPT_S  3
+
+/* SYSTEM_ENABLE_DOWNLOAD_G0CB_DECRYPT : R/W [2] : 1'b0 ; */
+
+#define SYSTEM_ENABLE_DOWNLOAD_G0CB_DECRYPT  (BIT(2))
+#define SYSTEM_ENABLE_DOWNLOAD_G0CB_DECRYPT_M  (BIT(2))
+#define SYSTEM_ENABLE_DOWNLOAD_G0CB_DECRYPT_V  0x1
+#define SYSTEM_ENABLE_DOWNLOAD_G0CB_DECRYPT_S  2
+
+/* SYSTEM_ENABLE_DOWNLOAD_DB_ENCRYPT : R/W [1] : 1'b0 ; */
+
+#define SYSTEM_ENABLE_DOWNLOAD_DB_ENCRYPT  (BIT(1))
+#define SYSTEM_ENABLE_DOWNLOAD_DB_ENCRYPT_M  (BIT(1))
+#define SYSTEM_ENABLE_DOWNLOAD_DB_ENCRYPT_V  0x1
+#define SYSTEM_ENABLE_DOWNLOAD_DB_ENCRYPT_S  1
+
+/* SYSTEM_ENABLE_SPI_MANUAL_ENCRYPT : R/W [0] : 1'b0 ; */
+
+#define SYSTEM_ENABLE_SPI_MANUAL_ENCRYPT  (BIT(0))
+#define SYSTEM_ENABLE_SPI_MANUAL_ENCRYPT_M  (BIT(0))
+#define SYSTEM_ENABLE_SPI_MANUAL_ENCRYPT_V  0x1
+#define SYSTEM_ENABLE_SPI_MANUAL_ENCRYPT_S  0
+
+#define SYSTEM_RTC_FASTMEM_CONFIG_REG          (DR_REG_SYSTEM_BASE + 0x048)
+
+/* SYSTEM_RTC_MEM_CRC_FINISH : RO [31] : 1'b0 ; */
+
+#define SYSTEM_RTC_MEM_CRC_FINISH  (BIT(31))
+#define SYSTEM_RTC_MEM_CRC_FINISH_M  (BIT(31))
+#define SYSTEM_RTC_MEM_CRC_FINISH_V  0x1
+#define SYSTEM_RTC_MEM_CRC_FINISH_S  31
+
+/* SYSTEM_RTC_MEM_CRC_LEN : R/W [30:20] : 11'h7ff ; */
+
+#define SYSTEM_RTC_MEM_CRC_LEN  0x000007ff
+#define SYSTEM_RTC_MEM_CRC_LEN_M  ((SYSTEM_RTC_MEM_CRC_LEN_V)<<(SYSTEM_RTC_MEM_CRC_LEN_S))
```

Comment:
The `<<` operator should have spaces around it. Also, the second set of parenthesis are not needed as the values are 
safely defined in the same place.

Reply:
Also, ideally, the definitions for these macros should be aligned.

Reply:
I fixed all of these and all the alignments.